alacritty_terminal/src/term/mod.rs

```diff
@@ -1870,6 +1870,20 @@ impl ansi::Handler for Term {
         self.color_modified[index] = true;
     }
 
+    /// Get the foreground/background color
+    #[inline]
+    fn request_special_color<W: io::Write>(&mut self, writer: &mut W, code: u8, index: usize) {
```

Comment:
In (most) of the other methods we add some trace output at the start of the handler functions. I think it would be worthwhile to do the same here.

Reply:
Added in a trace [here](https://github.com/rbong/alacritty/blob/fc3f86eded9af7b45b18c7a8354dda8e3ff93daf/alacritty_terminal/src/term/mod.rs#L1876)